### DISCUSSION TITLE
Question: What database does minio use in the minio server to store general information like object list and other things on dashboard

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [minio/minio#19980] [Q&A] [answered]
What database does minio use in the minio server to store general information like object list and other things on dashboard like users list, buckets list, groups, settings

**Comment by harshavardhana** [comment, marked as answer]
We do not use any database

**Reply by 0xMALVEE** [reply]
is it on a KV store or straight to disk?

**Reply by harshavardhana** [reply]
all data exists as objects on the drives.
